### DISCUSSION TITLE
Cannot consume internal events published to amq.rabbitmq.events for number of bindings

### DISCUSSION THREAD

**Discussion by eliyosef578** [rabbitmq/rabbitmq-server#12257] [Other]
Following on from the previous discussion (#12053)
if i try to load the def.json with only exchange.created bindings, the events reached to the events queue.
[def.json](https://github.com/user-attachments/files/16932751/def.json)
if i try to load the definitions.json with all the 4 bindings the events are not reached to the events queue.
[definitions.json](https://github.com/user-attachments/files/16932756/definitions.json)
this is the rabbit log : 
[rabbit.log](https://github.com/user-attachments/files/16932748/rabbit.log)

**Comment by michaelklishin** [comment]
@eliyosef578 the internal event mechanism was never meant to be used to track e.g. definition import progress. It is meant to be used for continuous monitoring of key events, for audit purposes.

Reasons can vary widely, including "the internal event mechanism needs to be initialized on boot". We will not tweak when that happens, leave alone when definitions are imported.

Please use the internal event mechanism as it was designed to be used. Monitoring definition import progress on boot was never the objective.
